source/Octopus.Server.Client/Model/DeploymentFreezes/DeleteDeploymentFreezeResponse.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Octopus.Client.Model.DeploymentFreezes;
+
+public class DeleteDeploymentFreezeResponse
+{
+
```

Comment:
We say we return a task but this returns nothing, am I missing something here?
Is this something the Async client takes care of for us?

Reply:
Thanks Dom, the comment is incorrect!